src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -698,7 +699,8 @@ prepareTxInfoRaw moneySource dstDistr = do
         coins = foldr1 unsafeAddCoin $ snd <$> dstDistr
     distr@(remaining, spendings) <- selectSrcAccounts coins notDstAccounts
     logDebug $ buildDistribution distr
-    bootEra <- gsIsBootstrapEra
+    epoch <- siEpoch <$> getCurrentSlotBlocking
```

Comment:
I used `getCurrentSlotBlocking` here, but I don't know whether it's okay to block here.